script/codesign/get-trusted-identity.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+
+valid_certs=$(security find-identity -p codesigning -v)
+if [[ $valid_certs == *"1)"* ]]; then
+  first_valid_cert=$(echo $valid_certs | sed 's/ \".*//' | sed 's/.* //')
+  echo $first_valid_cert
+  exit 0
+else
+  # No Certificate
+  exit 0
```

Comment:
maybe `exit 1`?

Reply:
`exit 1` causes things like `execSync` to throw instead of returning.  I exit with code `0` here to avoid having to try catch and instead inspect the output (like we would have to anyway)